renuo-bin-check.gemspec

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'renuo/bin-check/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'renuo-bin-check'
+  spec.version = Renuo::BinCheck::VERSION
+  spec.authors = ['Zora Fuchs']
+  spec.email = ['zora.fuchs@renuo.ch']
+
+  spec.summary = 'The Renuo bin-check automates running programms to check code quality of a rails application'
```

Comment:
actually not only rails apps, but any app 😉 
i would also append the following: caching and parallel execution